### PR TITLE
Add e2e-libvirt jobs to release-4.5 and add tests to release-4.6 for cluster-api-provider-libvirt

### PR DESCRIPTION
**Pull request by Prashanth684**
The non-generated jobs were not carried over to 4.5. Adding it back and also adding back the tests to 4.6

### CODE REVIEW

**Comment by Prashanth684** [comment]
@praveenkumar @cfergeau @sdodson can i get an approval for this? the 4.5 job fails because of https://github.com/openshift/cluster-api-provider-libvirt/pull/211

File: ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml [hunk 1]
```diff
@@ -222,6 +222,9 @@ objects:
         # gcloud compute will use this key rather than create a new one
         cp -a /tmp/cluster/ssh-privatekey ~/.ssh/google_compute_engine
         cp -a /tmp/cluster/ssh-publickey ~/.ssh/google_compute_engine.pub
+        echo 'ServerAliveInterval 30' | tee -a "${HOME}"/.ssh/config
+        echo 'ServerAliveCountMax 1200' | tee -a "${HOME}"/.ssh/config
+        chmod 0600 "${HOME}"/.ssh/config
```
Comment (cfergeau) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml:227]: This seems unrelated to what the commit log is saying.

Comment (praveenkumar) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml:227]: @cfergeau yes this should be in separate commit since without this change the server timeout happen.

Comment (Prashanth684) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml:227]: ah..yes ..good point...i'll put it in a separate commit
